etcdmain/gateway.go

```diff
@@ -77,6 +78,23 @@ func newGatewayStartCommand() *cobra.Command {
 	return &cmd
 }
 
+func stripSchema(eps []string) []string {
```

Comment:
@bdudelsack  heyitsanthony is right about not having the unit test. The test path should be covered in e2e test where the correct behavior gateway should be verified. Sorry for the wrong suggestion.